src/pyenphase/auth.py

```diff
@@ -18,7 +18,7 @@ class EnvoyAuth:
 
     def __init__(self, host: str) -> None:
         """Initialize the EnvoyAuth class."""
-        pass
+        pass  # pragma: no cover
```

Comment:
```suggestion
```

Reply:
This looks like an unnecessary pass statement

Reply:
can very well be, it was there originally, so didn´t know if it was there for a purpose. If no use we can remove it.

Reply:
Maybe auto generated. AFAICT its not needed

Reply:
Removed it. Tests and my Envoy still happy without it.